docs/DeveloperGuide.md

```diff
@@ -672,55 +571,55 @@ otherwise)
 
   Use case ends.
 
-* 3a. The given name is invalid.
-
+* 3a. The given index is invalid.
     * 3a1. SETA shows an error message.
 
       Use case resumes at step 2.
-
+    
 ****
 
-
 **Use case: Add help tag**
 
 **MSS**
 
-1. User requests to add help tag to a specific student.
+1. User requests to add help tag to a specific student in the list.
 2. SETA adds a help tag to that student.
 
    Use case ends.
 
 **Extensions**
 
-* 1a. Invalid or non-existent student name.
+* 1a. The given index is invalid.
     * 1a1. SETA shows an error message.
 
       Use case ends.
 
 ****
 
-
-**Use case: Delete a student**
+**Use case: Find student(s)**
 
 **MSS**
 
-1. User requests to list students.
-2. SETA shows a list of students.
-3. User requests to delete a specific student in the list.
-4. SETA deletes the person.
-
-   Use case ends.
+1. User requests to find a student or student(s) with the specific keyword.
```

Comment:
I see you have used 'keyword(s)' in the next line. Would it be more accurate and consistent if 'keyword' is changed to 'keyword(s)'?

Reply:
resolved in recent commit